docs/examples-and-tutorials/integrations/index.md

```diff
@@ -23,7 +23,6 @@ sidebar_position: 4
 ## Executor Plugins
 
 - [Kubectl](https://docs.botkube.io/usage/executor/kubectl): Easily execute Kubernetes commands in your channels with Botkube's Kubectl integration.
-- [Open AI](https://botkube.io/integration/chatgpt-botkube-kubernetes-integration): Harness the power of Open AI with Botkube's Doctor plugin for intelligent interactions and enhanced automation in your Kubernetes environment.
```

Comment:
I think it would be great to replace the link with one for AI assistant. Please ask Blair or Maria for the link and update it here 👍

Reply:
I can do that, but I'd think we could add current AI docs when we do the task of updating the docs.

Reply:
I do agree, but in the meantime can you add the link? https://botkube.io/blog/level-up-your-sre-workflow-automating-manual-tasks-with-botkube-ai-assistant

(Remember about changing that for 1.11 too 👍)